modules/Hoot/ui/modals/exportData.js

```diff
@@ -222,8 +222,8 @@ export default class ExportData {
                 includehoottags: self.includeHootTagsCheckbox.property( 'checked' ),
                 outputname: self.getOutputName(),
                 outputtype: self.getOutputType(),
-                tagoverrides: {},
-                textstatus: false,
+                // tagoverrides: "{}",
+                // textstatus: false,
```

Comment:
remove

Reply:
I'm gonna leave those, but will add a comment that they are unimplemented, i.e. they are hardcoded props for features from hoot1 that didn't make it into hoot2x yet.